course/resources/blurb-rqm.md

```diff
@@ -0,0 +1,3 @@
+We are an interdisciplinary group of graduate students interested in launching a third or fourth year undergraduate course in Reproducible Quantitative Methods (RQM). The curriculum is based on an [open curriculum](https://cbahlai.github.io/rqm-template/) developed by Dr. Christie Bahlai at Michigan State University through a Mozilla Science Lab fellowship. It is specifically designed to meet a growing demand for reproducible, openly accessible, statistically thorough, and well documented science. Although daunting at times, this approach to research presents an opportunity to improve the scientific practice, encourage broader participation in the scientific enterprise, and facilitate meeting the increasingly strict requirements from funding agencies in terms of data accessibility and reproducibility.
+
+The RQM course takes an approach that both produces conventional research products and trains students to work in accordance to sound practices within the framework laid out above. We will be teaching a pilot version of this course as a seminar topic in the department of Ecology and Evolutionary Biology (EEB) during the upcoming fall semester. If the pilot offering is positively met, we aim to offer a full version of the course the upcoming academic year, preferably as a cross-departmental effort between EEB and the department of Computer Science, and we are looking for a professor interested in championing this initiative.
```

Comment:
Maybe "interested in sponsoring" or "endorsing" rather than "champion". Championing suggests they would put a lot of work into it and advocating more for it. We are already doing that, plus it may make it seem like we need someone to do more work, which a busy professor may not be able to provide. Thoughts?